Formula/openldap.rb

```diff
@@ -25,6 +25,10 @@ class Openldap < Formula
 
   depends_on "openssl@1.1"
 
+  on_ventura :or_newer do
+    depends_on "groff" => :build
+  end
```

Comment:
If we're going to start doing this we may as well add `until:` to `uses_from_macos`.

Reply:
For runtime dependencies, an `until` would not be very safe. But it's ok for build-only.